manager/controllers/filter.go

```diff
@@ -115,7 +116,20 @@ func transformValues(fieldName string, originalValues []string) (transformedValu
 			transformedValues = append(transformedValues, originalValue)
 		}
 	}
-	return transformedValues
+
+	if len(transformedOperator) == len(originalOperator) {
```

Comment:
what situation is solving this 'if'?
I see transformedOperator has no value here.

Reply:
Ah, thanks. I used operator instead of values. This if detects whether "other" filter was used or not. If it was used we need to adjust operator type accordingly.